source/stdx/allocator/package.d

```diff
@@ -223,6 +223,10 @@ Source: $(PHOBOSSRC std/experimental/_allocator)
 
 module stdx.allocator;
 
+version (D_BetterC) {} else version = HasDRuntime;
+
+version (HasDRuntime):
```

Comment:
I am still unsure about this one. Basically I am versioning out the complete `package.d` in betterC. This means `make` and other allocator helper functions won't be available in betterC. Although the file also contains interfaces and classes, `make` and friends might just work in betterC. So I have the feeling I am hitting a fly with a hammer.

I don't care about it myself much, but maybe someone will complain later on.

Reply:
Considering as this library is used, i.e with DUB, we can merge and take fixups. Until a new tag os pushed everything is possible.